package.json

```diff
@@ -50,15 +50,13 @@
     "just-left-pad": "^2.1.0",
     "lodash.throttle": "^4.1.1",
     "mapbox-gl": "^2.0.1",
-    "node-sass": "^6.0.1",
-    "npm": "^7.7.6",
     "polka": "^1.0.0-next.20",
     "shpjs": "^3.6.3",
-    "sirv": "^1.0.0",
-    "svelte-inview": "^2.0.1"
+    "sirv": "^1.0.0"
   },
   "devDependencies": {
     "@beyonk/svelte-carousel": "^2.8.0",
```

Comment:
`@beyonk/svelte-carousel` this can be deleted. It was being used on the landing page in a older design